narratives/ncov-19B_resurgence.md

```diff
@@ -0,0 +1,192 @@
+Here we are highlighting another subclade in 19B with multiple spike mutations.
+While this clade does not have a D614G substitution, it instead has a substitution in the adjacent residue, Q613H.
+
+The subclade also contains F157L, V367F, and P681R.
+P681H, a different but still positively charged amino acid at the same location, is present in 501Y.V1.
+In a deep mutational scanning experiment of the receptor binding domain by [Starr et al](https://jbloomlab.github.io/SARS-CoV-2-RBD_DMS/), V367F was associated with increased expression.
+
+E484K (yellow) emerged within this subclade.
+This mutation is present in 501Y.V2 and 501Y.V3.
+Multiple preprints have demonstrated reduced neutralization to convalescent sera, monoclonal antibodies and sera from individuals with Moderna or Pfizer-BioNTech vaccines with this mutation ([Greaney et al](https://www.biorxiv.org/content/10.1101/2020.12.31.425021v1), [Liu et al](https://www.biorxiv.org/content/10.1101/2020.11.06.372037v1), and [Wang et al](https://www.biorxiv.org/content/10.1101/2020.12.31.425021v1)).
+
+R190S, a mutation present in 501Y.V3, occurs on the branch leading up to E484K.
+
+<!-- There is NO right-side text on this slide -->
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- SLIDE 7 -->
+# [Multiple emergences of L452R](https://nextstrain.org/groups/blab/ncov/19B?c=gt-S_452&d=tree&f_clade_membership=19B&p=full&label=clade:19B&legend=open)
+<!-- This is the left-side text -->
+Coloring by amino acid at spike 452, we observe three independent emergences of 452R mutations (blue) and one emergence of a 452M mutation (yellow). L452R has been observed in a [lineage growing in frequency in California]((https://www.cdph.ca.gov/Programs/OPA/Pages/NR21-020.aspx) but the biological impact of this mutation, if any, is unknown.
+
+One emergence of 452R was in 19B:501Y on a 614D backbone.
+The second emergence was on a 614N backbone along with spike R190S, a mutation also present in 501Y.V3
+The third emergence occurred on 614G backbone.
+This lineage also contains a spike 140-143 deletions. A 143-144 deletion is observed in 501Y.V1.
+
+These independent appearances of 452R could be examples of convergent evolution, which would suggest this mutation boosts viral fitness.
+
+<!-- There is NO right-side text on this slide -->
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- SLIDE 8 -->
+# [Closing summary](https://nextstrain.org/groups/blab/ncov/19B?d=tree,map,entropy&legend=open)
+
+<!-- This is the left-side text -->
+In this report, we explored the genetic composition of viruses currently circulating in clade 19B.
+We observed many 'familiar' mutations, including multiple emergences of spike D614G.
+
+We found the following spike mutations also present in a variant of concern (501Y.V1, 501Y.V2, and 501Y.V3):
+- 18F
+- 69-70 del
+- 143 del (along with 140-142 del)
+- 190S
+- 484K
+- 501Y (and 501T)
+- 614G (and 614N)
+- 655Y
+- 681R (681P is present in 501Y.V1)
+
```

Comment:
I notice in this list we don't mention 677 which you had in one of your original lists? I don't know if there's a legit reason it got left off, but if not, let's stick it in! As we've not noticed at least 7 independent origins of it in the USA and more globally. Completely unsure what it might mean but might be worth sticking in? Preprint hopefully out soon!

Reply:
I'd be happy to list others out if you think it's worthwhile. The entire R-hand side could be mutations with summary text on the left. I definitely felt like I was cherry picking on what other mutations to choose, so decided against. But we could list all spike mutations.

Reply:
I'm pretty easy on this. I'm happy to leave it how it is for the moment, and if we end up doing a little preprint or further work in any way, we can include at a later point (we'll actually be able to cite the 677 preprint then, anyway! Just got submitted yesterday so won't be up for a few days!)